pladder/misc.py

```diff
@@ -41,6 +42,20 @@ def capify(text):
     return text.title()
 
 
+def suspektify(text):
```

Comment:
The "varargs" flag in register_command is badly named. It should probably be named "join_args" or something like that. In your case the code can be simplified a bit by not using it.

If you want your command to receive a variable number for words, then you can simply skip the "varargs" flag and rewrite the top of suspektify like this:

    def suspektify(*words):

Note that words will be a tuple instead of a list, but I don't think that makes any difference in this function.